app/Jobs/UpdateWikiSiteStatsJob.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace App\Jobs;
+
+use App\Wiki;
+use App\WikiSiteStats;
+use Illuminate\Support\Facades\Http;
+use Illuminate\Support\Facades\Log;
+use Illuminate\Contracts\Queue\ShouldBeUnique;
+
+class UpdateWikiSiteStatsJob extends Job implements ShouldBeUnique
+{
+    public $timeout = 3600;
```

Comment:
I don't know if 1hr is enough to run the job in prod, some wikis in prod are huge. No need to change it now but could be something we need to observe

Reply:
We just poll the stats, so this is relatively cheap. If we ever hit the hour, we're doing something wrong.